smelt.lic

```diff
@@ -2,20 +2,12 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#smelt
 =end
 
-custom_require.call(%w[common common-crafting equipmanager])
+custom_require.call(%w[common common-crafting common-items])
 
 class Smelt
-  include DRC
-  include DRCC
 
   def initialize
-    EquipmentManager.new.empty_hands
-
-    settings = get_settings
-    @bag = settings.crafting_container
-    @bag_items = settings.crafting_items_in_container
-    @belt = settings.forging_belt
-    @rod = settings.forging_tools.find { |item| /rod/ =~ item }
+    DRCI.stow_hands
```

Comment:
Should this be moved to be after the args parsing? I.e. in case the args are wrong, we don't empty their hands